tenseal/cpp/context/tensealcontext.h

```diff
@@ -222,6 +234,11 @@ class TenSEALContext {
      * @returns true if the contexts are identical.
      **/
     bool equals(const std::shared_ptr<TenSEALContext>& other);
+    /**
+     * @returns a pointer to the threadpool dispatcher
+     **/
+    shared_ptr<sync::ThreadPool> dispatcher() { return _dispatcher; }
+    size_t get_concurrency() { return _threads.value_or(1); }
```

Comment:
So the default won't be to check hardware concurrency?

Reply:
I'll fix this, I mixed them up